pkg/bootstrap/config_test.go

```diff
@@ -92,6 +93,16 @@ func TestGetNodeMetaData(t *testing.T) {
 	t.Setenv(IstioMetaPrefix+"OWNER", inputOwner)
 	t.Setenv(IstioMetaPrefix+"WORKLOAD_NAME", inputWorkloadName)
 
+	// prepare a pod label file
+	os.MkdirAll("./etc/istio/pod/", os.ModePerm)
```

Comment:
can use t.TempDir and os.chdir

Reply:
done

Reply:
doesn't seem pushed?

Reply:
pushed, it create a tmp dire and changed dir, this is to create `/etc/istio/pod/` under the team dir.